AEMG/training.py

```diff
@@ -47,7 +47,9 @@ def __init__(self, config, loaders, verbose):
 
         self.reset_losses()
 
-        self.criterion = nn.MSELoss(reduction='mean')
+        self.dynamics_criterion = nn.MSELoss(reduction='mean')
+        self.labels_criterion = nn.TripletMarginLoss()
```

Comment:
`reduction='mean'`?

Reply:
It's the default value, but I can still add it